marketplace/optimizely/src/AppPage/ContentTypes.js

```diff
@@ -131,14 +133,14 @@ export default function ContentTypes({
   };
 
   return (
-    <div className="f36-margin-top--xl">
+    <div className={styles.body}>
```

Comment:
Instead of using classes here you can just wrap all components in `Typography` and will be padded automatically

Reply:
I changed this element to `Typography`. However, I did have to keep some spacing on the `Button` below.

Reply:
yep, right. `Button` is not a part of Typography components.